### DISCUSSION TITLE
Season 2 pvp gear

### DISCUSSION THREAD

**Discussion by tbcxd** [chromiecraft/chromiecraft#7622] [Suggestions]
Season 3 had been going for 1month&half and many people are getting their pieces of season 3 pvp gear from doing soloq or other arena.

The gap between a player that is just starting out and someone who has been playing since the season start is getting bigger and bigger as season3 pvp gear is a very big upgrade for many classes and specs.

We will get to a point where it will be very difficult for a player geared in s1 to stand any chance vs someone wearing full s3 gear& some BT pve gear, regardless of how good the s1 geared player is, thus discouraging new players to join the already not-so-active pvp scene.

Given the circumstances I wrote above, I would like to suggest that we add the season 2 pvp gear for honor

This will also give many of us new reasons to get back into doing battlegrounds(right now people only get their daily 20 arena points win and stop queuing for the day)

I think I can speak on behalf of 90% of the pvp community when I say that we want things to be easier for us and everyone else that is considering joining the pvp scene.

Many of us were expecting the s2 to be available for honor as soon as s3 started and we were quite disappointed to hear it was not the case this season.

For those reading this, please DO post a comment of what you think about this suggestion and don't just thumbsup it. 

Thank you for reading.

**Comment by Apathyxia** [comment]
I agree with this suggestion. However, I would also like to add that S1 gear should be made available with gold should S2 gear be made available with honour. As stated, the difference in gear is significant. If S1 gear was available for gold, this would easily give more people a fair chance to even try BGs when they instantly don't feel outgeared if they are wearing at best few blue pieces from reputation vendors. This would also encourage queueing up with alts when getting even the minimum "acceptable" gear is feasible. 

As the current activity in BGs is quite low outside of Call-to-Arms weekends, in a 4v4 match one person with blue gear almost always guarantees a loss for their team, and makes it unmotivating to even try to basically play a 3v4 game.

**Reply by vlad7394** [reply]
I second the statements above.
